src/translations/translationHelper.js

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash';
+
+export const toObject = (body) => {
+  const object = {};
+  const lines = body.split(/\n/);
+  lines.forEach((line) => {
+    const message = line.split(':');
+    if (message.length === 2) {
+      object[message[0].trim()] = message[1].trim();
+    }
+  });
+  return object;
+};
+
+export const getMessages = (replies) => {
+  const localesMap = {};
+  const messages = {};
+  const sortedReplies = _.sortBy(replies, 'net_rshares');
+
+  Object.keys(sortedReplies).forEach((key) => {
+    const reply = sortedReplies[key];
+    if (reply.depth === 1 && reply.body.length === 4 && reply.body.substring(0, 2) === '# ') {
```

Comment:
for conditions that are long and confusing like this, it's better to give them a name and use the name inside the if for easier readability:
```javascript
// not sure if name fits the action or not just example
const isTranslationCommentAvailable = (reply.depth === 1 && reply.body.length === 4 && reply.body.substring(0, 2) === '# ');

if (isTranslationCommentAvailable) {
  ...
}
```